### PR TITLE
#8 Add lists section with trackBy/key examples.

### LINKED ISSUE #8: Add 'Lists' section
At least a simple example of using lists:

- [X] **AngularJS:**
`ng-repeat`, `trackby`

- [X] **Angular:**
`*ngFor `

- [X] **React:**
`key`

Preferably also with links to docs.

### PR DESCRIPTION
**Pull request by vitaliy-bobrov**
Hi, I've created basic examples for repeating lists with trackBy/key functionality, let me know about any additions I need to add. fixes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -397,14 +398,124 @@ constructor(props) {
...
+  }
+
+  render() {
+    const books = this.state.books;
+
+    return (
+      <ul>
+        books.map(book => {
+          return (
+            <li key="id">
```
Comment (feimosi) [inline README.md]: Should be `key={ book .id }`

File: README.md [hunk 2]
```diff
@@ -397,14 +398,124 @@ constructor(props) {
...
+      ]
+    };
+  }
+
+  render() {
+    const books = this.state.books;
+
+    return (
+      <ul>
+        books.map(book => {
```
Comment (feimosi) [inline README.md]: `books.map` should be inside `{ }`

File: README.md [hunk 3]
```diff
@@ -397,14 +398,124 @@ constructor(props) {
...
+          id: 3,
+          title: "JavaScript: The Definitive Guide",
+          author: "David Flanagan"
+        }
+      ]
+    };
+  }
+
+  render() {
+    const books = this.state.books;
```
Comment (feimosi) [inline README.md]: Can also be `const { books } = this.state;`

File: README.md [hunk 4]
```diff
@@ -397,14 +398,124 @@ constructor(props) {
...
+      id: 2,
+      title: "JavaScript: The Good Parts",
+      author: "Douglas Crockford"
+    },
+    {
+      id: 3,
+      title: "JavaScript: The Definitive Guide",
+      author: "David Flanagan"
+    }
+  ];
```
Comment (feimosi) [inline README.md:478]: Just add type definition as it's in TypeScript;

**Review by feimosi — CHANGES_REQUESTED** [review]
Great, thanks!

File: README.md (hunk 4 above)
Comment (vitaliy-bobrov) [reply README.md:478]: Added interface example.

File: README.md (hunk 1 above)
Comment (vitaliy-bobrov) [reply README.md]: Fixed, sorry, most of time using ng4 in the last project, React no so often.

**Comment by vitaliy-bobrov** [comment]
Implemented changes and added links to the official docs.

**Comment by feimosi** [comment]
Looks good, thanks for the contribution!

**Review by feimosi — APPROVED** [review]

**Pull request merged**